unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneController.cs

```diff
@@ -916,5 +920,22 @@ public void ReIntegrateIsolatedScene()
 
         private bool IsPortableExperienceAlreadyConfirmed(string pxId) =>
             confirmedExperiencesRepository.Contains(pxId);
+
+        private bool IsPortableExperienceInWhiteList(string pxId)
+        {
+            FeatureFlag flags = DataStore.i.featureFlags.flags.Get();
+            FeatureFlagVariantPayload payload = flags.GetFeatureFlagVariantPayload("initial_portable_experiences:calendarpx");
```

Comment:
this is doing a request for every portable experience, or you fetch this only one time and store it on that DataStore variable ?

Reply:
It is in cache, not querying to an external service